packages/flutter/lib/src/material/app.dart

```diff
@@ -263,11 +263,6 @@ class MaterialApp extends StatefulWidget {
 
 class _MaterialScrollBehavior extends ScrollBehavior {
```

Comment:
Let me know if you would like to have ScrollBehavior accept an optional 
accentColor argument so that _MaterialScrollBehavior can be removed.